Update the Admonitions section (#2348)
tarantool/doc: doc/contributing/docs/markup/admonitions.rst

        For example, in the description of :doc:`/reference/reference_lua/box_session/on_connect`
        trigger we warn a user about some consequences of his actions.

*   Important:

    ..  code-block:: rst

>         ..  IMPORTANT::

Reviewer: ```suggestion
        ..  important::
```

Reviewer: There's no need to use capital letters.

>     ..  IMPORTANT::

Reviewer: ```suggestion
    ..  important::
```

        This block contains essential information that the user should know while doing something.

>         For example, the block :doc:`here </getting_started/getting_started_go>` is used to warn the user against

Reviewer: ```suggestion
        For example, the block :doc:`in Getting Started with Docker </getting_started/getting_started_go>` is used to warn the user against
```

Reviewer: https://www.tarantool.io/en/doc/latest/contributing/docs/style/#specify-a-link-text
